rootex/common/common.h

```diff
@@ -3,6 +3,8 @@
 #include "common/types.h"
 #include "os/os.h"
 #include "script/interpreter.h"
+#include <vector>
+#include <sstream>
```

Comment:
We don't include the commonly used `std` types directly like this. They all have been defined in common/types.h. Sorry, I actually meant to say to make changes in `common/types.h` instead of `common/common.h`.

Reply:
Also the function can be moved in there too